### PR TITLE
Update/restructure code

### PR DESCRIPTION
**Pull request by mugren**
Refactored code into multiple modules for easier comprehending, and extending.
Besides sanity checks and running the app, the changes (endpoints) were not tested before pushing.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,120 +1,30 @@
...
-app.get("/l/:publicId", async (req, res) => {
-  const shorturl = req.params.publicId;
-  try {
-    const checkObject = await URL.findOne({ shortURL: shorturl });
-    res.redirect(checkObject.longURL);
-  } catch (err) {
-    res.send(err);
-  }
-});
+apiRoutes(app)
```
Comment (nodus8tollens) [inline index.js:26]: What is going on here and how does it work?

Comment (nodus8tollens) [reply index.js:26]: I see that you've refactored the routes into a single function but I can't really envision how it operates in terms of the function (apiRoutes) calling the Express server in index.js (apiRoutes(app)).

**Review by nodus8tollens — APPROVED** [review]
Everything works as imagined, both in terms of req/res management and DB storage.

File: index.js (hunk 1 above)
Comment (mugren) [reply index.js:26]: It's the same as before, except that I moved that 'iterative' part of defining the routes into a function. 
In JS you can pass any type of object to a function, so I pass reference of the express app instance ('app'), and then using that reference to define the routes in the routes function.

It the api-routes.js there is the export part `module.exports = routes` which means that I export the routes function as default from that module. Hence calling this one `const apiRoutes = require('./src/api-routes')` will actually give me the reference to the default fn exported, or this `const routes = function(app) {`

**Pull request merged**